src/lando/main/logging.py

```diff
@@ -68,6 +66,9 @@ def __init__(self, *args, mozlog_logger=None, **kwargs):
     def format(self, record):
         """Formats a log record and serializes to mozlog json"""
 
+        # NOTE: Django passes some fields that are not JSON serializable in the record
+        # (for example, the WSGIRequest object representing the request). Therefore
+        # those values are converted to a string to avoid any issues when serializing.
```

Comment:
As @cgsheeh pointed out, it may be nicer/cleaner in the future to use something like https://github.com/mozilla-services/python-dockerflow to cover this and other cases. It might also have additional improvements/requirements that aren't included in this legacy code.